### PR TITLE
Add Makefile to cki_lib

### PR DESCRIPTION
**Pull request by ftjahjad**
Add Makefile skeleton to cki_lib.

### CODE REVIEW

**Comment by ftjahjad** [comment]
@cki-bot , Please test

**Comment by ftjahjad** [comment]
@rasibley , and @jbastian  , please review a change. :smiley:

File: cki_lib/Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+run:
+	echo nothing > /dev/null
+
+METADATA=testinfo.desc
+
+$(METADATA): Makefile
+	@touch $(METADATA)
```
Comment (emanchado) [inline cki_lib/Makefile]: Why `touch` if the next line has `> $(METADATA)`?

Comment (jbastian) [reply cki_lib/Makefile]: Good question! It was just copied/pasted from other Makefiles in the tree. This pattern is in pretty much every Makefile for Beaker tasks and has been around for many years.

**Comment by rasibley** [comment]
Inspecting podman test in the job results, I no longer see the Makefile error in the harness.log:

```
** Installing dependencies
** Extracting cki_lib/
** Extracting cki_lib/Makefile
** Extracting cki_lib/libcki.sh
use_pty:FALSE make testinfo.desc
use_pty:FALSE rstrnt-package install podman-tests podman
```

File: cki_lib/Makefile (hunk 1 above)
Comment (rasibley) [reply cki_lib/Makefile]: @emanchado just an fyi, next phase to remove restraint-rhts is to convert all the Makefile's to use a metadata file instead. 
https://restraint.readthedocs.io/en/latest/tasks.html#metadata

**Pull request merged**